src/script/components/calling/FullscreenVideoCall.styles.ts

```diff
@@ -87,4 +87,5 @@ export const paginationButtonStyles: CSSObject = {
   position: 'absolute',
   top: 'calc(50% - 75px)',
   width: 56,
+  zIndex: 1,
```

Comment:
matching the z-index of a focused tile:
![image](https://github.com/wireapp/wire-webapp/assets/78490891/f1d15ed9-6de6-4d2c-aed4-2fbe3368fadb)